pkg/metrics/stats.go

```diff
@@ -134,4 +137,28 @@ func InitStatsMetrics() {
 		Name:      "register_task",
 		Help:      "gauge of plan replayer registered task",
 	})
+	StatsDeltaLoadHistogram = NewHistogram(
+		prometheus.HistogramOpts{
+			Namespace: "tidb",
+			Subsystem: "statistics",
+			Name:      "stats_delta_load_duration_seconds",
+			Help:      "Bucketed histogram of processing time for the background statistics loading job",
+			Buckets:   prometheus.ExponentialBuckets(0.01, 2, 24), // 10ms ~ 24h
+		},
+	)
+	StatsDeltaUpdateHistogram = NewHistogram(
+		prometheus.HistogramOpts{
+			Namespace: "tidb",
+			Subsystem: "statistics",
+			Name:      "stats_delta_update_duration_seconds",
+			Help:      "Bucketed histogram of processing time for the background stats_meta update job",
+			Buckets:   prometheus.ExponentialBuckets(0.01, 2, 24), // 10ms ~ 24h
+		},
+	)
+	StatsLoadObjectGauge = NewGaugeVec(prometheus.GaugeOpts{
```

Comment:
any usage of this one

Reply:
Just a try and found that it's very well. deleted.